attributes/default.rb

```diff
@@ -108,6 +108,10 @@
 when 'gentoo'
   default['ntp']['leapfile'] = "#{node['ntp']['varlibdir']}/ntp.leapseconds"
 when 'solaris2'
+  if node['platform_version'] < '5.11'
+    default['ntp']['packages'] = %w(SUNWntpu SUNWntpr)
+    default['ntp']['pkg_source'] = '/cdrom/Solaris_10/Product'
```

Comment:
The `solaris_package` resource requires a `source` directive on Solaris 10. The documentation currently says the `source` property is optional -- however it is a bug in the docs for an open PR to correct: https://github.com/chef/chef-web-docs/pull/713